model/generate.go

```diff
@@ -495,7 +514,8 @@ func (g *GeneratedProject) findTasksAndVariantsWithSpecificActivations(requester
 		batchTimeTasks := []string{}
 		for _, bvt := range bv.Tasks {
 			if isStepbackTask(g.Task, bv.Name, bvt.Name) {
-				res.stepbackTasks[bv.Name] = append(res.stepbackTasks[bv.Name], bvt.Name)
+				stepbackInfo := specificStepbackInfo{task: bvt.Name, depth: g.Task.StepbackDepth + 1}
```

Comment:
Is it worth adding a note that depth + 1 is intended only for new tasks being created by stepback? Or can we move the `+ 1` part to [here](https://github.com/evergreen-ci/evergreen/blob/211ebdf33de10de18951e2ed3adcfb1a3ad870a6/model/lifecycle.go#L1247) where it's more immediately relevant?

Reply:
I think it's clear enough here since we have the line `isStepbackTask` right above and I'd rather what's passed in via this info is the depth to be used, but I can add a comment?

Reply:
I think it's clear enough here since we have the line `isStepbackTask` right above that this is only intended for stepback; however, I think it's actually a mistake to use +1 here since generated tasks should have the same stepback depth as their generator (since this isn't a deeper stepback).

Reply:
Tested this behavior ^ and confirmed shouldn't add +1